theano/gpuarray/opt.py

```diff
@@ -743,7 +740,7 @@ def local_gpua_specifyShape(op, context_name, inputs, outputs):
 @register_opt('fast_compile')
 @op_lifter([theano.compile.ops.Shape])
 @register_opt2([tensor.compile.ops.Shape], 'fast_compile')
-def local_gpua_shape(node, context_name, inputs, outputs):
+def local_gpua_shape(op, context_name, inputs, outputs):
```

Comment:
Missed it! But there seems to be some problem with this opt in transferring nodes.